src/helpers/greetings.js

```diff
@@ -1,6 +1,7 @@
-const GREETINGS = ["Hey", "Hi", "Hello", "Morning", "Afternoon", "Evening", "Greetings"];
+const GREETINGS = ["Hello", "Morning", "Afternoon", "Evening", "Greetings"];
```

Comment:
I think a better approach will be to only listen to authorized Telegram handles' messages. Or a simpler implementation might be to check group permissions/if is admin?

Alternatively, it could operate slowly bearing rate limits in mind?